website/src/documentation/dsls/sql/calcite/aggregate-functions.md

```diff
@@ -18,11 +19,13 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Beam SQL: Aggregate functions
+# Beam SQL aggregate functions for Calcite
```

Comment:
can we standardize on a name format for all of the pages in a given section? (except the overview - made a suggestion in another comment)

Beam SQL aggregate functions (Calcite)